src/java/MilitaryAirTransport.java

```diff
@@ -0,0 +1,37 @@
+package java;
+
+public class MilitaryAirTransport extends AirTransport {
+    boolean ejectionSystems;
+    int numberOfMissiles;
+
+    public MilitaryAirTransport(int power, int maksSpeed, int weight, String carModel, int wingSpan, int minLength,
+                                boolean ejectionSystems,int numberOfMissiles) {
+        super(power, maksSpeed, weight, carModel, wingSpan, minLength);
+        this.ejectionSystems=ejectionSystems;
+        this.numberOfMissiles=numberOfMissiles;
+    }
+
+    @Override
+    void Output() {
+        super.Output();
+        if (ejectionSystems){
+        System.out.println("Ejection Systems: yes ");
+        }
+        System.out.println("Number of Missiles: " + numberOfMissiles);
+    }
+    void shot() {
```

Comment:
@Hlieb А вот между методами лучше оставлять 1 пустую строку
```suggestion
    }
    
    void shot() {
```